decidim-proposals/app/helpers/decidim/proposals/application_helper.rb

```diff
@@ -17,10 +17,13 @@ module ApplicationHelper
       #
       # Returns a String.
       def humanize_proposal_state(state)
-        value = if state == "accepted"
+        value = case state
```

Comment:
This method smells. Can you use i18n defaults?

http://guides.rubyonrails.org/i18n.html#defaults

Something like `I18n.t(value, scope: "decidim.proposals.answers", default: :not_answered)` should do the trick